### ISSUE TITLE
Cannot get http://maia.usno.navy.mil/ser7/deltat.data

### ISSUE THREAD

**Issue by Deuchnord** [Deuchnord/kosmorro#1]
The program fails at downloading the data necessary to compute ephemeris with the error:

```
OSError: cannot get http://maia.usno.navy.mil/ser7/deltat.data because <urlopen error [Errno -2] Name or service not known>
```

This error comes from the Skyfield package and is reported here: skyfielders/python-skyfield#296.

**Comment by Deuchnord** [comment]
In the meantime, required missing files can be downloaded here: https://drop.deuchnord.fr/r/nc_yVZ54#kftMk1IlEQ0YgKNnyhOw+auwuCpEqn7Y45Ee1dXVQ48= (link valid until December 1st, 2019 09:10 AM UTC).

Extract the TAR.GZ and put them in the `cache` folder at the root of the project (if the folder doesn't exist, create it).

**Comment by Deuchnord** [comment]
Fixed in #4
